bin/test_runner.rb

```diff
@@ -286,10 +292,9 @@ def check_todo_list
   def run
     ####  main running loop
     while true
```

Comment:
As discussed, this method of `while true` is used to make it easier for you to change it to `while false` instead of commenting out with `=begin..=end`.  Let's add that as a comment above.